src/main/clm_initializeMod.F90

```diff
@@ -471,6 +475,10 @@ subroutine initialize2( )
        if ( use_c13 .and. use_c13_timeseries ) then
           call C13_init_TimeSeries()
        end if
+    ! use_fates_spitfire is assigned an integer value in the namelist
+    ! see bld/namelist_files/namelist_definition_clm4_5.xml for details
+    else if (use_fates_spitfire > 1) then
+       call bgc_vegetation_inst%Init2(bounds_proc, NLFilename)
```

Comment:
Don't call into bgc_veg you want to use the instance that's tied to FATES

Reply:
You point to fates_fire_data_method in clmfates_interface

Reply:
In this case I don't understand. Unless I also need to make a copy of
`subroutine Init2`
in clmfates_interface, which seems redundant to me.

Reply:
The reasoning is actually similar to above, but even stronger here, because there are things in the BGC VegetationFacade that you do NOT want for the FATES case. The only think you want is the FATES Fire data Init, you don't want DNPhenologyInit nor SoilBiogeochemCompetitionInit (because it must already be called separately for FATES).

Reply:
@ekluzek pls let me know if I misunderstood your explanation in the next commit that I'm about to push.

Reply:
I added the subroutines `Init2` and `InterpFileInputs` to both `FATESFireDataMod` and `FATESFireNoDataMod`. I made the latter empty for consistency with the other subroutines, although the NoData versions do not get called because of the if statement `else if (use_fates_spitfire > 1)`. I see two options:
1) Delete the NoData copies of `Init2` and `InterpFileInputs`
2) Change the if statement to `else if (use_fates)`